docs: add third-party storage docs (#728)
harvester/docs: docs/vm/edit-vm.md

You can add additional volumes to the VM after booting. You can also expand the size of the volume after shutting down the VM, click the VM and go to the `Volumes` tab, edit the size of the expanded volume. After restarting the VM and waiting for the resize to complete, your disk will automatically finish expanding.

:::note

> If using third party storage, you need to ensure that the third-party storage provider supports volume expansion before resizing your volume.

Reviewer: ```suggestion
If you are using external storage, ensure that the storage provider supports volume expansion before you resize volumes.
```

> Since the third party storage is supported. You need to ensure the expansion capability of the third-party storage provider before you resize the volume.

Reviewer 2: ```suggestion
If using third party storage, you need to ensure that the third-party storage provider supports volume expansion before resizing your volume.
```
